docs/docs/recipes.md

```diff
@@ -770,6 +770,100 @@ export const pageQuery = graphql`
 - [Guide to creating pages from data programmatically](/docs/programmatically-create-pages-from-data/)
 - [Example repo](https://github.com/gatsbyjs/gatsby/tree/master/examples/recipe-sourcing-markdown) for this recipe
 
+### Sourcing from Wordpress
+
+#### Prerequisites
+
+- An existing [Gatsby site](/docs/quick-start/) with a `gatsby-config.js` and `gatsby-node.js` file
+- An existing template component such as `src/templates/post.js`
```

Comment:
Should be also show the content or link to a file to see what is in the `templates/post.js`?

Reply:
That does sound helpful! We could add a link to the full-length Reference guide or a blog post as a note. They can refer to it for the template content if they wish.  
This is so that the scope of the recipe is kept brief.
Thoughts?

Reply:
@gatsbyjs/docs and @marcysutton knows a best practice for the receipts

Reply:
Awaiting the review from her then

Reply:
I think the best way forward is to create an example for the recipe. Here's some precedent for a Markdown version: https://github.com/gatsbyjs/gatsby/tree/master/examples/recipe-sourcing-markdown

With some code in place that is made intentionally for this recipe, we can link to the `post.js` file in that example in the prerequisites AND give people something to play around with. The [`using-wordpress`](https://github.com/gatsbyjs/gatsby/tree/master/examples/using-wordpress) example is a bit more complicated as it uses Advanced Custom Fields, otherwise I would have recommended that.

Reply:
That would be great! I have opened a PR for an example [here](/gatsbyjs/gatsby/pull/17425). We can link this once it is approved to give the people more clarity.